tests/ZendTest/Filter/StringToLowerTest.php

```diff
@@ -177,4 +177,16 @@ public function testReturnUnfiltered($input)
     {
         $this->assertEquals($input, $this->_filter->filter($input));
     }
+
+    /**
+     * @group 7147
+     */
+    public function testFilterUsesGetEncodingMethod()
+    {
+        $filterMock = $this->getMock('\Zend\Filter\StringToLower', array('getEncoding'));
```

Comment:
Note that in strings you always deal with an absolute namespace, therefore the leading `\` is not needed and should be avoided.